docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+      tokenUrl: "https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/oauth2/v2.0/token"
+      jwksUrl: "https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/discovery/v2.0/keys"
+      identity:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://identity.camunda.example.com"
+        initialClaimName: "email"
+        initialClaimValue: test.user@camunda.com
+      optimize:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://optimize.camunda.example.com"
+      core:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        tokenScope: "00000000-0000-0000-0000-000000000000/.default"
+        redirectUrl: "https://core.camunda.example.com"
+      console:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        wellKnown: https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/v2.0/.well-known/openid-configuration
+        audience: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        redirectUrl: "https://console.camunda.example.com"
+      webModeler:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        clientApiAudience: "00000000-0000-0000-0000-000000000000"
+        publicApiAudience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://modeler.camunda.example.com"
+```
+
+If you would like some more guidance relating to authentication, then please refer to the following guides:
+
+- [Connect to an OpenID Connect provider](/self-managed/setup/guides/connect-to-an-oidc-provider.md)
+
+### Connect External Databases
+
+The next stage of the production setup is configuring databases. To make it easy for testing, the Camunda Helm Chart provides external, dependency Helm Charts for Databases such as Bitnami Elasticsearch Helm Chart and Bitnami PostgresQL Helm Chart. Within a production setting, these dependency charts should be disabled and production databases should be used instead. For example, instead of the Bitnami Elasticsearch dependency chart, we will use Amazon OpenSearch, and instead of the Bitnami PostgreSQL dependency chart, we will use Amazon Aurora PostgreSQL.
+
+In our scenario the Core component, and the Optimize importer communicates with a singular Amazon OpenSearch instance. On the other hand, the identity and web-modeler component are connected to seperate Amazon Aurora PostgreSQL instances.
+
+It is assumed that you already have one Amazon OpenSearch instance and two Amazon Aurora PostgreSQL instances setup and ready to go with a username, password, and URL.
```

Comment:
why are we proposing two Postgresql instances?
What's the advantage to a single instance with two individual databases inside?

Reply:
In https://docs.camunda.io/docs/next/self-managed/setup/deploy/amazon/amazon-eks/eks-terraform/#configure-the-database-and-associated-access we use one Postgres instance for example and as mentioned just add databases within the Postgres instance, so data is still separated from each other, also each app their own user / pw etc.

If there's a good reason on why one should go with 2 separate instances, we should consider aligning those.

Reply:
That's a good point! This is why I wanted a review from infraex team to get feedback like this. Thanks!

Reply:
I will mention only one aws aurora psql instance instaed